docs-template/EXAMPLE_README.md.tpl

```diff
@@ -240,6 +240,14 @@ Bevy support for WebGPU is being worked on, but is currently experimental.
 
 To build for WebGPU, you'll need to disable default features and add all those you need, making sure to omit the `webgl2` feature.
 
+WebGPU depends on unstable APIs so you will also need to pass the `web_sys_unstable_apis` flag to your builds. For example:
+
+```sh
+RUSTFLAGS=--cfg=web_sys_unstable_apis cargo run ...
```

Comment:
```suggestion
RUSTFLAGS=--cfg=web_sys_unstable_apis cargo build ...
```
`cargo run` doesn't work without extra config for wasm

Reply:
Changed in 2f1d56656443b6b8886fec14d0682ade3a8b375c